state/observable/SirenSummonAction.js

```diff
@@ -81,11 +81,15 @@ export class SirenSummonAction extends Routable(SirenAction) {
 	_updateAction() {
 		this.action = {
 			has: true,
-			summon: async(observables) => {
+			summon: async(observables, bypassCache) => {
 				this._prepareAction(observables);
 				this._readyToSend = true;
-				// todo: we should be able to just return what we get back from fetch, but it is returning the json
-				await fetch(this);
+
+				const response = await fetch(this);
```

Comment:
```suggestion
				const response = await fetch(this, bypassCache);
```
I'm fairly confident that the routedState is set by the call to `fetch`. I'm surprised that the tests work 🤔 https://github.com/BrightspaceHypermediaComponents/foundation-engine/blob/master/state/fetch.js#L16

Reply:
You should be able to just pass the flag to fetch and leave everything else as is

Reply:
I see what you're saying, yeah. I've passed `true` to `fetch` like you noted above, but it actually still returns the cached result. I will figure out what's going on.

Reply:
Follow up: Investigated and discussed this with Maya and Zina, and edited the PR based on our  conclusions